corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -6,19 +6,6 @@
 <!--[if IE 8]><html lang="{{ LANGUAGE_CODE }}" class="lt-ie9"><![endif]-->
 <!--[if gt IE 8]><!--><html lang="{{ LANGUAGE_CODE }}"><!--<![endif]-->
   <head>
-    {% if ANALYTICS_IDS.GTM_ID %}
-      {# This is fine as an inline script; it's independent of all HQ code and all third-party libraries #}
-      <!-- Google Tag Manager -->
-      <script>
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-          new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-          j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-          'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','{{ ANALYTICS_IDS.GTM_ID }}');
-      </script>
```

Comment:
where is this happening now? or its not needed anymore?

same for iframe that was removed later?

Reply:
How do we confirm this does not break anything that was being tracked already?

Reply:
Check in with SaaS tech about this.

Reply:
I was able to confirm that Google Tag manager is not used at present by checking below things

- No option in Cloud PR to add `GTM_ID` without which above snippet is not executed.
- Checked through browser tools that `gtm.js` is not loaded for Prod or India.

I think it is wise to double check with SAAS though.

Reply:
Hey @mkangia , I did checked with SAAS about this. They do not use it and suggested to check with Marketing. 
I talked with Marketing they do use Google Tag Manager, I am just waiting for more details on it, should be able to get it in next couple of days.

Reply:
**Update**: We reached out to marketing and they do have a Google Tag manager account with a container for `CommcareHQ`. As expected, the container for CommcareHQ is empty and not currently being used and aligns with the current findings.

(Just for FYI - they have a container for `www.dimagi.com` which is currently in use)

Reply:
So it is safe to remove this section or not?

Reply:
Yes it is. It is safe even it was being used as the change just loads the same file from the javascript instead of the html. 
I think it is still wise to merge this only after the decision to use GTM is finalised. 
(No use to have the code that is not actively used or planned to be used.)